ex13.py

```diff
@@ -0,0 +1,13 @@
+from sys import argv
```

Comment:
see the list of different libraries available in python. Also list popular libraries here.